libraries/AP_Volz_Protocol/AP_Volz_Protocol.h

```diff
@@ -48,6 +48,9 @@
 #define VOLZ_EXTENDED_POSITION_CENTER 		0x0800	// Extended Position Data Format defines 0 as 0x0800 - decimal 2048
 #define VOLZ_EXTENDED_POSITION_MAX 			0x0F80	// Extended Position Data Format defines +100 as 0x0F80 decimal 3968 -> full range decimal 3840
 
+#define PWM_POSITION_MIN 					1000
```

Comment:
We should probably name these definitions starting with VOLZ_.  So perhaps VOLZ_PWM_MIN and VOLZ_PWM_MAX.  Aleternatively move them to the .cpp file to keep their scope small but I think from a readers point of view it is nice to have all the definitions together.